packages/datetime/src/dateRangePicker.tsx

```diff
@@ -48,6 +48,12 @@ export interface IDateRangePickerProps extends IDatePickerBaseProps, IProps {
     defaultValue?: DateRange;
 
     /**
+     * Constrains months to be sequential
+     * @default true
+     */
+    isSequential?: boolean;
```

Comment:
This effect of this prop isn't immediately clear from its name, and we could also afford to improve the description. Maybe something like this:

```
/**
 * Whether to keep displayed months contiguous in the calendar.
 * If false, each side of the calendar can move independently to non-contiguous months.
 * @default true
 */
keepDisplayMonthsContiguous?: boolean;
```

?